src/style.css

```
@@ -1 +1,107 @@
-/* add styles here */
+@font-face{
+  font-family: Avenir;
+  src:url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face{
+  font-family: Avenir;
+  src:url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+:root {
+  --basic-color: #3d4e61;
+}
+
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+
+input {
+  font-family: inherit;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body{
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+ }
+
+.search-bar{
+  margin-left: 8px;
+  margin-right: 8px;
+  position: relative;
+}
+
+.search-bar__big{
+  margin: 20px 8px;
+}
+
+.search-bar__label{
+  position: relative;
+}
+
+.search-bar__label::before{
+  content: '';
+  position: absolute;
+  background-image: url(./images/Search.svg);
+  z-index: 10;
+}
+
+.search-bar__label--big::before{
+  width: 19px;
+  height: 19px;
+  top: 1px;
+  left: 26px;
+}
+
+.search-bar__input{
+  width: 100%;
+  position: relative;
+  padding-top: 23px;
+  padding-bottom: 23px;
+  padding-left: 62px;
+  color: var(--basic-color);
+  border: 1px solid #E1E7ED;
+  box-shadow: 0 1px 8px 0 #3D4E611A;
+  font-size: 16px;
+  border-radius: 3px;
+}
+
+.search-bar__input:hover{
+  box-shadow: 0 3px 8px 0 #3D4E6133;
+}
+
+.search-bar__input:focus{
+  font-weight: 900;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  outline:none;
+  border-radius: 3px;
+}
```

```suggestion
.search-bar__input:focus {
  font-weight: 900;
  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
  outline:none;
  border-radius: 3px;
}
```

check please css styles ordeting
https://9elements.com/css-rule-order/